packages/theming/theme-types/src/Color.types.ts

```diff
@@ -385,7 +385,7 @@ export interface ControlColorTokens {
   ghostDisabledBorder: ColorValue;
   ghostDisabledContent: ColorValue;
   ghostDisabledIcon: ColorValue;
-
+  /* Deprecated */
```

Comment:
Can we actually put this is as a comment on top of every token so it shows up in intelligence? Similar to line 317 and above